/mnt/hwfile/huqinghao/miniconda3/bin/python
2024-05-09 02:46:07,330 - INFO - Note: detected 128 virtual cores but NumExpr set to maximum of 64, check "NUMEXPR_MAX_THREADS" environment variable.
2024-05-09 02:46:07,330 - INFO - Note: NumExpr detected 128 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2024-05-09 02:46:07,330 - INFO - NumExpr defaulting to 8 threads.
2024-05-09 02:46:08,833	INFO scripts.py:1182 -- Did not find any active Ray processes.

==========
== CUDA ==
==========

CUDA Version 12.1.0

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

*************************
** DEPRECATION NOTICE! **
*************************
THIS IMAGE IS DEPRECATED and is scheduled for DELETION.
    https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/support-policy.md

Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d85b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
/usr/local/lib/python3.10/dist-packages/pydantic/_internal/_fields.py:160: UserWarning: Field "model_name_or_path" has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
  warnings.warn(
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9330>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-09 02:46:23 api_server.py:202] vLLM API server version 0.3.3
INFO 05-09 02:46:23 api_server.py:203] args: Namespace(host=None, port=8000, uvicorn_log_level='info', allow_credentials=False, allowed_origins=['*'], allowed_methods=['*'], allowed_headers=['*'], api_key=None, served_model_name=None, lora_modules=[], delta_modules=[Delta(name='delta-1', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-1'), Delta(name='delta-2', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-2'), Delta(name='delta-3', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-3'), Delta(name='delta-4', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-4'), Delta(name='delta-5', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-5'), Delta(name='delta-6', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-6'), Delta(name='delta-7', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-7'), Delta(name='delta-8', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-8')], swap_modules=[], chat_template=None, response_role='assistant', ssl_keyfile=None, ssl_certfile=None, ssl_ca_certs=None, ssl_cert_reqs=0, root_path=None, middleware=[], model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer=None, revision=None, code_revision=None, tokenizer_revision=None, tokenizer_mode='auto', trust_remote_code=False, download_dir=None, load_format='auto', dtype='auto', kv_cache_dtype='auto', max_model_len=2048, worker_use_ray=False, pipeline_parallel_size=1, tensor_parallel_size=4, max_parallel_loading_workers=None, ray_workers_use_nsight=False, block_size=16, enable_prefix_caching=False, seed=0, swap_space=4, gpu_memory_utilization=0.8, max_num_batched_tokens=None, max_num_seqs=256, max_logprobs=5, disable_log_stats=False, quantization=None, enforce_eager=True, max_context_len_to_capture=8192, disable_custom_all_reduce=False, tokenizer_pool_size=0, tokenizer_pool_type='ray', tokenizer_pool_extra_config=None, enable_lora=False, max_loras=1, max_lora_rank=64, lora_extra_vocab_size=256, lora_dtype='auto', max_cpu_loras=32, enable_delta=True, max_deltas=8, max_cpu_deltas=32, max_delta_bitwidth=4, device='auto', image_input_type=None, image_token_id=None, image_input_shape=None, image_feature_size=None, scheduler_delay_factor=0.0, enable_prefetch=True, scheduler_policy='deltaserve', engine_use_ray=False, disable_log_requests=True, max_log_len=None)
2024-05-09 02:46:26,918	INFO worker.py:1749 -- Started a local Ray instance.
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9c00>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-09 02:46:29 llm_engine.py:87] Initializing an LLM engine (v0.3.3) with config: model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.float16, lora_config=None, delta_config=DeltaConfig(max_deltas=8, max_bitwidth=4, delta_dtype=torch.int8, max_cpu_deltas=32, delta_extra_vocab_size=0, pack_factor=Fraction(2, 1), kernel=<QuantKernel.TRITON: 'triton'>), max_seq_len=2048, download_dir=None, load_format=auto, tensor_parallel_size=4, disable_custom_all_reduce=False, quantization=None, enforce_eager=True, kv_cache_dtype=auto, device_config=cuda, seed=0)
INFO 05-09 02:46:29 llm_engine.py:237] ignored verifying delta_config
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9cc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d93f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d80d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da4d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dada0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dace0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da410>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8040>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8e20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9ed0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db670>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9990>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d90c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8790>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da800>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dada0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dad10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da8c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8400>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d95a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db340>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d87c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d80a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da290>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dab60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dada0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbd00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db070>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da9e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da110>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8dc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db4f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9210>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8520>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da6b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db9a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbcd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9c90>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dba90>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da500>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8190>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9360>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbf40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8610>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dabf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db1f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dadd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399b7df0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9e70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db460>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399daa10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da830>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da0e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8b20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da140>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399daaa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db820>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dada0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Using bitblas
WARNING 05-09 02:56:26 models.py:48] Using bitblas delta modules
INFO 05-09 02:56:26 selector.py:16] Using FlashAttention backend.
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbf10>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=29523)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f13b854cdc0>
[36m(RayWorkerVllm pid=29523)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=29523)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=29523)[0m     self.remove()
[36m(RayWorkerVllm pid=29523)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=29523)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=29523)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=29241)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7fb200118dc0>
[36m(RayWorkerVllm pid=29241)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=29241)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=29241)[0m     self.remove()
[36m(RayWorkerVllm pid=29241)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=29241)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=29241)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=29523)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f13b854cdc0>
[36m(RayWorkerVllm pid=29523)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=29523)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=29523)[0m     self.remove()
[36m(RayWorkerVllm pid=29523)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=29523)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=29523)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9540>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbd30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db970>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dad70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da1d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a0880>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dafe0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbac0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbb80>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9e70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a0370>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d99c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9b70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db5b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da470>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9000>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a0970>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da830>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399daa10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db460>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbf10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a0ac0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbd60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db010>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db910>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da020>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da200>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a1180>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da4a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da4d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=29523)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f13b854cdc0>
[36m(RayWorkerVllm pid=29523)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=29523)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=29523)[0m     self.remove()
[36m(RayWorkerVllm pid=29523)[0m   File "/usr/local/lib/python3.10/dist-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=29523)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=29523)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db280>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db490>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a0700>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbf70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbf10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db460>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399daa10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da080>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9120>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8340>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dae00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dae60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9ea0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da350>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9ff0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9840>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9870>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399daf50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dac20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8250>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da7d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9fc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9540>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbd60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399b7a60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9810>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d93f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9c60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db880>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da7a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a1270>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9f00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbc70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9720>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da350>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a1630>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db760>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8220>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d93c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db520>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399daf80>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a0970>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9f00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbc70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9720>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399da350>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a0c40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9c00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d8220>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d93c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db520>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dad40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55395a1a80>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db910>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399db010>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=29091)[0m Using bitblas
[36m(RayWorkerVllm pid=29241)[0m Using bitblas
[36m(RayWorkerVllm pid=29523)[0m Using bitblas
[36m(RayWorkerVllm pid=29241)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=29241)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=29523)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=29523)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=29523)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=29523)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=29523)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=29523)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/b954930a6e09bfea9bc32b3ce9aa2405dc619cb024945ef61fc0f44afccdbc1a/tvm_rt_mod'
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:05:14 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:05:14 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:05:19 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:05:19 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:10:19 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:10:19 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:10:49 model_runner.py:155] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:10:49 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:10:49 model_runner.py:155] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:10:49 worker_manager.py:293] Starting prefetching thread
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:10:49 model_runner.py:155] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:10:49 worker_manager.py:293] Starting prefetching thread
INFO 05-09 03:10:49 model_runner.py:155] Loading model weights took 6.1114 GB
INFO 05-09 03:10:49 worker_manager.py:293] Starting prefetching thread
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399d9e70>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:10:52 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:10:52 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:10:52 quant_linear_triton.py:557] We should warm up the kernel here 
WARNING 05-09 03:10:53 quant_linear_triton.py:557] We should warm up the kernel here 
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f55399dbbe0>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-09 03:11:05 ray_gpu_executor.py:266] # GPU blocks: 13566, # CPU blocks: 1310
INFO 05-09 03:11:08 block_manager.py:264] disable automatic prefix caching
WARNING 05-09 03:11:08 serving_chat.py:373] No chat template provided. Chat API will not work.
INFO:     Started server process [19147]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     ::1:34124 - "GET /sysinfo HTTP/1.1" 200 OK
Translating from base-model to /vllm/.idea/full_models/Llama-2-13b-hf
Warming up starts
{'id': 22, 'prompt': 'USER: Write a python one line lambda function that calculates mean of two lists, without using any imported libraries. The entire function should fit on a single line, start with this. mean = lambda A:\nASSISTANT:', 'timestamp': 0, 'model': 'delta-4', 'min_tokens': 32, 'max_tokens': 32}
INFO 05-09 03:11:10 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
WARNING 05-09 03:11:10 worker_manager.py:335] Missed prefetch, fall back to loading delta 4 now
INFO 05-09 03:11:10 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:11:10 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:10 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:11:10 worker_manager.py:335] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:10 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:10 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:10 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:11:10 worker_manager.py:335] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:10 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:10 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:10 worker_manager.py:325] Adding delta 4 to prefetching queue: [4]
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:11:10 worker_manager.py:335] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:10 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:10 models.py:268] [main] Lossless Compression Disabled
INFO 05-09 03:11:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:11:26 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 15.551 seconds
INFO 05-09 03:11:26 worker_manager.py:353] Main thread loading delta 4 done
INFO 05-09 03:11:26 worker_manager.py:368] [1715195486.3282819] activating delta
INFO 05-09 03:11:26 worker_manager.py:373] [1715195486.435708] CPU -> GPU time: 0.1074
INFO 05-09 03:11:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:27 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 17.310 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:28 worker_manager.py:353] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:28 worker_manager.py:368] [1715195488.0806515] activating delta
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:28 worker_manager.py:373] [1715195488.221578] CPU -> GPU time: 0.1409
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:28 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 17.686 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:28 worker_manager.py:353] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:28 worker_manager.py:368] [1715195488.4595933] activating delta
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:28 worker_manager.py:373] [1715195488.5929291] CPU -> GPU time: 0.1333
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:29 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 18.573 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:29 worker_manager.py:353] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:29 worker_manager.py:368] [1715195489.3518157] activating delta
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:29 worker_manager.py:373] [1715195489.4829729] CPU -> GPU time: 0.1311
INFO 05-09 03:11:34 metrics.py:276] Avg prompt throughput: 9.6 tokens/s, Avg generation throughput: 3.5 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO:     ::1:34126 - "POST /v1/completions HTTP/1.1" 200 OK
Warming up ends
Issuing queries
sending 1 queries at 1.6
sending 1 queries at 4.2
sending 1 queries at 6.0
sending 1 queries at 7.6000000000000005
sending 1 queries at 8.700000000000001
sending 1 queries at 10.8
sending 1 queries at 11.9
sending 1 queries at 16.400000000000002
sending 1 queries at 23.0
sending 1 queries at 23.900000000000002
sending 1 queries at 27.1
sending 1 queries at 28.6
sending 1 queries at 30.3
sending 1 queries at 35.5
sending 1 queries at 35.6
sending 2 queries at 35.800000000000004
sending 1 queries at 39.400000000000006
sending 1 queries at 42.400000000000006
sending 1 queries at 46.5
sending 1 queries at 54.2
sending 1 queries at 57.400000000000006
sending 1 queries at 58.6
INFO 05-09 03:11:39 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
WARNING 05-09 03:11:39 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
INFO 05-09 03:11:39 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:11:39 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:39 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:11:39 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:39 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:39 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:39 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:11:39 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:39 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:39 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:39 worker_manager.py:325] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:11:39 worker_manager.py:335] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:39 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:39 models.py:268] [main] Lossless Compression Disabled
INFO 05-09 03:11:42 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
INFO 05-09 03:11:55 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 15.788 seconds
INFO 05-09 03:11:55 worker_manager.py:353] Main thread loading delta 6 done
INFO 05-09 03:11:55 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:11:55 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:56 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 17.024 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:56 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:58 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 18.697 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:58 worker_manager.py:353] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:59 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 19.241 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:59 worker_manager.py:353] Main thread loading delta 6 done
INFO 05-09 03:11:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.6 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:11:59 worker_manager.py:325] Adding delta 1 to prefetching queue: [8, 1]
WARNING 05-09 03:11:59 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
WARNING 05-09 03:11:59 worker_manager.py:344] Discarding prefetching delta 8
INFO 05-09 03:11:59 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:11:59 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:59 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:59 worker_manager.py:325] Adding delta 1 to prefetching queue: [8, 1]
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:11:59 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:59 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:11:59 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:59 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:59 worker_manager.py:325] Adding delta 1 to prefetching queue: [8, 1]
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:11:59 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:59 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:11:59 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:59 worker_manager.py:325] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:59 worker_manager.py:325] Adding delta 1 to prefetching queue: [8, 1]
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:11:59 worker_manager.py:335] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:59 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:11:59 models.py:268] [main] Lossless Compression Disabled
INFO 05-09 03:12:14 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 14.861 seconds
INFO 05-09 03:12:14 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 18.819 seconds
INFO 05-09 03:12:14 worker_manager.py:353] Main thread loading delta 8 done
INFO 05-09 03:12:14 worker_manager.py:278] Prefetching delta 8 done
WARNING 05-09 03:12:14 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:14 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 15.133 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:14 worker_manager.py:353] Main thread loading delta 8 done
INFO 05-09 03:12:14 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:12:14 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:14 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:14 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:12:16 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:12:16 worker_manager.py:344] Discarding prefetching delta 1
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:16 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:16 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:17 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 17.363 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:17 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:17 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:17 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:12:18 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:12:18 worker_manager.py:344] Discarding prefetching delta 1
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:18 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:18 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:19 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 19.489 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:19 worker_manager.py:353] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:19 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:19 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:12:22 worker_manager.py:335] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:12:22 worker_manager.py:344] Discarding prefetching delta 1
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:22 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:22 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:26 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 10.025 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:26 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 11.679 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:26 worker_manager.py:353] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:26 worker_manager.py:278] Prefetching delta 1 done
INFO 05-09 03:12:28 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 13.182 seconds
INFO 05-09 03:12:28 worker_manager.py:353] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:30 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 13.101 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:30 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 11.476 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:30 worker_manager.py:353] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:30 worker_manager.py:278] Prefetching delta 1 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:31 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 9.695 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:31 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 12.672 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:32 worker_manager.py:353] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:32 worker_manager.py:278] Prefetching delta 1 done
INFO 05-09 03:12:33 metrics.py:276] Avg prompt throughput: 0.6 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 7 to prefetching queue: [7]
INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 3 to prefetching queue: [7, 3]
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 7 to prefetching queue: [7]
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 3 to prefetching queue: [7, 3]
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 7 to prefetching queue: [7]
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 3 to prefetching queue: [7, 3]
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 7 to prefetching queue: [7]
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 3 to prefetching queue: [7, 3]
INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 2 to prefetching queue: [7, 3, 2]
WARNING 05-09 03:12:33 worker_manager.py:335] Missed prefetch, fall back to loading delta 2 now
INFO 05-09 03:12:33 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:12:33 models.py:268] [main] Lossless Compression Disabled
total threads: 23
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 2 to prefetching queue: [7, 3, 2]
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:12:33 worker_manager.py:335] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:33 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:33 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 2 to prefetching queue: [7, 3, 2]
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:12:33 worker_manager.py:335] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:33 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:33 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:33 worker_manager.py:325] Adding delta 2 to prefetching queue: [7, 3, 2]
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:12:33 worker_manager.py:335] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:33 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:33 models.py:268] [main] Lossless Compression Disabled
INFO 05-09 03:12:52 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 19.178 seconds
INFO 05-09 03:12:52 worker_manager.py:353] Main thread loading delta 2 done
INFO 05-09 03:12:52 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:12:52 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:53 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 19.999 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:53 worker_manager.py:353] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:53 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:53 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:53 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 20.272 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:53 worker_manager.py:353] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:53 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:53 models.py:268] [prefetching] Lossless Compression Disabled
WARNING 05-09 03:12:53 worker_manager.py:335] Missed prefetch, fall back to loading delta 3 now
INFO 05-09 03:12:53 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:12:53 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:54 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 21.040 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:54 worker_manager.py:353] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:54 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:54 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m WARNING 05-09 03:12:54 worker_manager.py:335] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:54 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:12:54 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:12:55 worker_manager.py:335] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:55 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:12:55 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m WARNING 05-09 03:12:56 worker_manager.py:335] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:56 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:12:56 models.py:268] [main] Lossless Compression Disabled
INFO 05-09 03:13:20 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 28.224 seconds
INFO 05-09 03:13:20 worker_manager.py:278] Prefetching delta 7 done
INFO 05-09 03:13:21 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 27.427 seconds
INFO 05-09 03:13:21 worker_manager.py:353] Main thread loading delta 3 done
INFO 05-09 03:13:21 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:13:21 models.py:268] [prefetching] Lossless Compression Disabled
INFO 05-09 03:13:21 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 0.317 seconds
INFO 05-09 03:13:21 worker_manager.py:278] Prefetching delta 3 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:25 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 31.222 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:25 worker_manager.py:278] Prefetching delta 7 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:27 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 30.997 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:27 worker_manager.py:353] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:27 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:27 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:31 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 36.729 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:31 worker_manager.py:353] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:13:32 worker_manager.py:335] Missed prefetch, fall back to loading delta 7 now
[36m(RayWorkerVllm pid=29523)[0m WARNING 05-09 03:13:32 worker_manager.py:344] Discarding prefetching delta 7
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:32 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:32 models.py:268] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:34 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 41.275 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:34 worker_manager.py:278] Prefetching delta 7 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:35 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 3.337 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:35 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 41.736 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:35 worker_manager.py:353] Main thread loading delta 7 done
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:35 worker_manager.py:278] Prefetching delta 7 done
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:35 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 41.002 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:35 worker_manager.py:353] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:35 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:35 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:36 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 0.851 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:36 worker_manager.py:278] Prefetching delta 3 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:36 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 9.185 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:37 worker_manager.py:278] Prefetching delta 3 done
INFO 05-09 03:13:37 metrics.py:276] Avg prompt throughput: 1.8 tokens/s, Avg generation throughput: 0.1 tokens/s, Running: 14 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:13:37 worker_manager.py:325] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:37 worker_manager.py:325] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:37 worker_manager.py:325] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:37 worker_manager.py:325] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:37 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:37 models.py:268] [prefetching] Lossless Compression Disabled
INFO 05-09 03:13:38 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-09 03:13:38 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:38 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:38 models.py:268] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:38 models.py:214] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:38 models.py:268] [prefetching] Lossless Compression Disabled
INFO 05-09 03:13:43 metrics.py:276] Avg prompt throughput: 72.7 tokens/s, Avg generation throughput: 9.5 tokens/s, Running: 22 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
INFO:     ::1:34308 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:13:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.5 tokens/s, Running: 21 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:54 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 15.231 seconds
[36m(RayWorkerVllm pid=29523)[0m INFO 05-09 03:13:54 worker_manager.py:278] Prefetching delta 5 done
INFO 05-09 03:13:54 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 15.511 seconds
INFO 05-09 03:13:54 worker_manager.py:278] Prefetching delta 5 done
INFO 05-09 03:13:55 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.9 tokens/s, Running: 21 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
INFO:     ::1:34290 - "POST /v1/completions HTTP/1.1" 200 OK
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:57 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 19.127 seconds
[36m(RayWorkerVllm pid=29091)[0m INFO 05-09 03:13:57 worker_manager.py:278] Prefetching delta 5 done
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:58 models.py:324] Disk -> CPU: Loaded 1674.62 MiB in 19.868 seconds
[36m(RayWorkerVllm pid=29241)[0m INFO 05-09 03:13:58 worker_manager.py:278] Prefetching delta 5 done
INFO 05-09 03:14:00 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.0 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:14:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.4 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:14:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.5 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:14:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.6 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO:     ::1:34390 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:34320 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:14:22 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.5 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-09 03:14:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.4 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 1 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO:     ::1:34288 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:34324 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:14:33 metrics.py:276] Avg prompt throughput: 3.8 tokens/s, Avg generation throughput: 19.2 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO:     ::1:34370 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:14:38 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.1 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-09 03:14:44 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.0 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.7%, CPU KV cache usage: 0.0%
INFO:     ::1:34229 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:14:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 16.7 tokens/s, Running: 15 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.7%, CPU KV cache usage: 0.0%
INFO:     ::1:34242 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:14:55 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.6 tokens/s, Running: 14 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.7%, CPU KV cache usage: 0.0%
INFO 05-09 03:15:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.1 tokens/s, Running: 14 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.7%, CPU KV cache usage: 0.0%
INFO 05-09 03:15:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.1 tokens/s, Running: 14 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.8%, CPU KV cache usage: 0.0%
INFO:     ::1:34312 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:15:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.9 tokens/s, Running: 13 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.7%, CPU KV cache usage: 0.0%
INFO 05-09 03:15:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 16.4 tokens/s, Running: 13 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.8%, CPU KV cache usage: 0.0%
INFO:     ::1:34352 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:34338 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:15:22 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.1 tokens/s, Running: 11 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.7%, CPU KV cache usage: 0.0%
INFO:     ::1:34248 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:34318 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:15:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.1 tokens/s, Running: 9 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO:     ::1:34306 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:15:33 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.2 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:15:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.1 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-09 03:15:44 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.2 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-09 03:15:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.2 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO:     ::1:34206 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:15:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.6 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:06 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 7 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO:     ::1:34196 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:16:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.2 tokens/s, Running: 6 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO:     ::1:34386 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:16:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.0 tokens/s, Running: 5 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.9 tokens/s, Running: 5 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO:     ::1:34218 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:16:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.2 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:36 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.3 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:41 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.4 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.4 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO:     ::1:34266 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:16:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.5 tokens/s, Running: 3 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
INFO 05-09 03:16:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.7 tokens/s, Running: 3 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.4%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:02 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.8 tokens/s, Running: 3 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO:     ::1:34332 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:17:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 5.8 tokens/s, Running: 2 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO:     ::1:34226 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:17:12 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 5.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:22 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:32 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:38 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:48 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-09 03:17:53 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO:     ::1:34322 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-09 03:18:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 1.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:18:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:18:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:18:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:18:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:18:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:19:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:19:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:19:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:19:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:19:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:19:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:20:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:20:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:20:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:20:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:20:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:20:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:21:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:21:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:21:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:21:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:21:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:21:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:22:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:22:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:22:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:22:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:22:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:22:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:23:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:23:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:23:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:23:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:23:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:23:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:24:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:24:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:24:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:24:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:24:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:24:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:25:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:25:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:25:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:25:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:25:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:25:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:26:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:26:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:26:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:26:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:26:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:26:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:27:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:27:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:27:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:27:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:27:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:27:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:28:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:28:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:28:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:28:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:28:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:28:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:29:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:29:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:29:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:29:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:29:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:29:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:30:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:30:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:30:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:30:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:30:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:30:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:31:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:31:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:31:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:31:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:31:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:31:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:32:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:32:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:32:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:32:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:32:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:32:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:33:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:33:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:33:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:33:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:33:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:33:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:34:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:34:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:34:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:34:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:34:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:34:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:35:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:35:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:35:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:35:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:35:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:35:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:36:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:36:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:36:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:36:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:36:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:36:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:37:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:37:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:37:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:37:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:37:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:37:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:38:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:38:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:38:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:38:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:38:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:38:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:39:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:39:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:39:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:39:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:39:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:39:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:40:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:40:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:40:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:40:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:40:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:40:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:41:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:41:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:41:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:41:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:41:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:41:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:42:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:42:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:42:27 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-09 03:42:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 0 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
slurmstepd: error: *** JOB 3668364 ON HOST-10-140-60-3 CANCELLED AT 2024-05-09T03:42:41 ***
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [19147]
Exception in callback functools.partial(<function _raise_exception_on_finish at 0x7f7aa9a56680>, error_callback=<bound method AsyncLLMEngine._error_callback of <vllm.engine.async_llm_engine.AsyncLLMEngine object at 0x7f7b49d50370>>)
handle: <Handle functools.partial(<function _raise_exception_on_finish at 0x7f7aa9a56680>, error_callback=<bound method AsyncLLMEngine._error_callback of <vllm.engine.async_llm_engine.AsyncLLMEngine object at 0x7f7b49d50370>>)>
Traceback (most recent call last):
  File "/usr/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "uvloop/loop.pyx", line 1511, in uvloop.loop.Loop.run_until_complete
  File "uvloop/loop.pyx", line 1504, in uvloop.loop.Loop.run_until_complete
  File "uvloop/loop.pyx", line 1377, in uvloop.loop.Loop.run_forever
  File "uvloop/loop.pyx", line 555, in uvloop.loop.Loop._run
  File "uvloop/loop.pyx", line 474, in uvloop.loop.Loop._on_idle
  File "uvloop/cbhandles.pyx", line 83, in uvloop.loop.Handle._run
  File "uvloop/cbhandles.pyx", line 63, in uvloop.loop.Handle._run
  File "/usr/local/lib/python3.10/dist-packages/uvicorn/server.py", line 68, in serve
    with self.capture_signals():
  File "/usr/lib/python3.10/contextlib.py", line 142, in __exit__
    next(self.gen)
  File "/usr/local/lib/python3.10/dist-packages/uvicorn/server.py", line 328, in capture_signals
    signal.raise_signal(captured_signal)
  File "/usr/local/lib/python3.10/dist-packages/ray/_private/worker.py", line 1414, in sigterm_handler
    sys.exit(signum)
SystemExit: 15

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/vllm/vllm/engine/async_llm_engine.py", line 525, in run_engine_loop
    await self._request_tracker.wait_for_new_requests()
  File "/vllm/vllm/engine/async_llm_engine.py", line 200, in wait_for_new_requests
    await self.new_requests_event.wait()
  File "/usr/lib/python3.10/asyncio/locks.py", line 214, in wait
    await fut
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "uvloop/cbhandles.pyx", line 63, in uvloop.loop.Handle._run
  File "/vllm/vllm/engine/async_llm_engine.py", line 52, in _raise_exception_on_finish
    task.result()
asyncio.exceptions.CancelledError
